meteomatics/tests/integration_test.rs

```diff
@@ -0,0 +1,785 @@
+    let df_s = CsvReader::new(file)
+        .infer_schema(Some(100))
+        .has_header(true)
+        .with_ignore_parser_errors(true)
+        .finish()
+        .unwrap();
+
+    // Query using rust connector
+    // Credentials
+    dotenv().ok();
+    let api_key: String = env::var("METEOMATICS_PW").unwrap();
+    let api_user: String = env::var("METEOMATICS_USER").unwrap();
+    
+    // Create API connector
+    let meteomatics_connector = APIClient::new(
+        api_user,
+        api_key,
+        10,
+    );
+
+    // Create time information
+    let start_date = Utc.ymd(1989, 11, 9).and_hms_micro(18, 0, 0, 0);
+    let end_date = start_date + Duration::days(1);
+    let interval = Duration::hours(12);
+
+    // Create Parameters
+    let parameters = vec![String::from("t_2m:C"), String::from("precip_1h:mm")];
+
+    // Create Locations
+    let p1: Point = Point { lat: 52.52, lon: 13.405};
+    let coords: Vec<Point> = vec![p1];
+
+    // Call endpoint
+    let df_q = meteomatics_connector
+        .query_time_series(&start_date, &end_date, &interval, &parameters, &coords, &None)
+        .await
+        .unwrap();
+    println!("Rust result: {:?}", df_q);
+    println!("Python result: {:?}", df_s);
+    assert!(df_s.frame_equal(&df_q));
+}
+
+#[tokio::test]
+async fn query_time_series_two_point_two_param() {
+    // Reference data from python connector
+    let s = r#"lat,lon,validdate,t_2m:C,precip_1h:mm
+    52.5,13.4,1989-11-09T18:00:00Z,6.8,0.05
+    52.5,13.4,1989-11-10T06:00:00Z,1.4,0.0
+    52.5,13.4,1989-11-10T18:00:00Z,5.4,0.0
+    52.4,13.5,1989-11-09T18:00:00Z,6.9,0.0
+    52.4,13.5,1989-11-10T06:00:00Z,1.5,0.0
+    52.4,13.5,1989-11-10T18:00:00Z,5.3,0.0
+    "#;
+    let file = Cursor::new(s);
+    let df_s = CsvReader::new(file)
+        .infer_schema(Some(100))
+        .has_header(true)
+        .with_ignore_parser_errors(true)
+        .finish()
+        .unwrap();
+
+    // Query using rust connector
+    // Credentials
+    dotenv().ok();
+    let api_key: String = env::var("METEOMATICS_PW").unwrap();
+    let api_user: String = env::var("METEOMATICS_USER").unwrap();
+    
+    // Create API connector
+    let meteomatics_connector = APIClient::new(
+        api_user,
+        api_key,
+        10,
+    );
+
+    // Create time information
+    let start_date = Utc.ymd(1989, 11, 9).and_hms_micro(18, 0, 0, 0);
+    let end_date = start_date + Duration::days(1);
+    let interval = Duration::hours(12);
+
+    // Create Parameters
+    let parameters = vec![String::from("t_2m:C"), String::from("precip_1h:mm")];
+
+    // Create Locations
+    let p1: Point = Point { lat: 52.50, lon: 13.40};
+    let p2: Point = Point { lat: 52.40, lon: 13.50};
+    let coords: Vec<Point> = vec![p1, p2];
+
+    // Call endpoint
+    let df_q = meteomatics_connector
+        .query_time_series(&start_date, &end_date, &interval, &parameters, &coords, &None)
+        .await
+        .unwrap();
+    println!("Rust result: {:?}", df_q);
```

Comment:
I would remove printlines from tests since they contribute nothing. In many testing frameworks in other languages outputting to console out is not permitted.